Userland/Applications/PixelPaint/Image.h

```diff
@@ -74,6 +75,8 @@ class Image : public RefCounted<Image> {
 private:
     explicit Image(const Gfx::IntSize&);
 
+    static RefPtr<Image> create_from_pixel_paint_file(const String& file_path);
```

Comment:
To comply with [the recent change in project code style](https://github.com/SerenityOS/serenity/commit/dd9b8ee7ef5666b0ba257a40c7c024628f821448), please use east-const.